PWGCF/FemtoDream/femtoDreamProducerTask.cxx

```diff
@@ -166,20 +167,24 @@ struct femtoDreamProducerTask {
     v0Cuts.init<aod::femtodreamparticle::ParticleType::kV0, aod::femtodreamparticle::ParticleType::kV0Child, aod::femtodreamparticle::cutContainerType>(&qaRegistry);
   }
 
-  void process(aod::FilteredFullCollision const& col,
+  void process(soa::Join<aod::Collisions, aod::EvSels, aod::Mults>::iterator const& col,
                aod::FilteredFullTracks const& tracks,
                o2::aod::V0Datas const& fullV0s) /// \todo with FilteredFullV0s
   {
     /// First thing to do is to check whether the basic event selection criteria are fulfilled
     if (!colCuts.isSelected(col)) {
+      // if it is a trigger run, store unselected collision as well
+      if (ConfIsTrigger) {
+        outputCollision(col.posZ(), col.multV0M(), colCuts.computeSphericity(col, tracks)); // globalIndexAO2D is required for the 3-body trigger
```

Comment:
Also here the comment can go because you don't have that column anymore

Reply:
Here the trigger is for the rows, not columns. To explain, that if it is a trigger run, even the events which are not selected are saved in the table. I will update the comment to make it more clear and explain in both situations

Reply:
I don't mind a comment in general, but bugs me is `globalIndexAO2D`. This what the old column name which you have removed. So the comment should be "store all collisions for 3-body trigger" or so... Maybe I am missing something

Reply:
I misunderstood which for which line the comment was meant. I will fix this. Sorry